agent/agentscripts/agentscripts.go

```diff
@@ -80,6 +80,21 @@ func New(opts Options) *Runner {
 
 type ScriptCompletedFunc func(context.Context, *proto.WorkspaceAgentScriptCompletedRequest) (*proto.WorkspaceAgentScriptCompletedResponse, error)
 
+type runnerScript struct {
+	runOnPostStart bool
```

Comment:
suggestion: this is fine for now, but I could see this being an option on `WorkspaceAgentScript` in future.